generate.sh

```diff
@@ -5,12 +5,34 @@ set -e
 
 EXAMPLES_GENERATED_DIR=examples/generated
 
+# Determine which platform we're running on
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     platform=Linux;;
+    Darwin*)    platform=Mac;;
+    *)          platform="UNKNOWN:${unameOut}"
+esac
+echo "You appear to be running on ${platform}"
+
 echo "Ensuring we have NPM packages installed..."
 npm install
 
 echo "Compiling ts-protoc-gen..."
 npm run build
 
+echo "Downloading protoc v3.5.1... for ${platform}"
```

Comment:
Suggest we read the version from a variable to make it easier to upgrade

Reply:
Good shout! Fixed.